external/emulation/emulation/masks.py

```diff
@@ -40,14 +40,28 @@ def __call__(self, state: FortranState, emulator: FortranState) -> FortranState:
 
 
 class LevelMask:
-    def __init__(self, key: str, start: Optional[int], stop: Optional[int]):
+    def __init__(
+        self,
+        key: str,
+        start: Optional[int],
+        stop: Optional[int],
+        fill_value: Union[float, str, None] = None,
+    ):
         self.key = key
         self.start = start
         self.stop = stop
+        self.fill_value = fill_value
 
     def __call__(self, state: FortranState, emulator: FortranState) -> FortranState:
         use_fortran_state = slice(self.start, self.stop)
         # Fortran state TOA is index 79, and dims are [z, sample]
         emulator_field = np.copy(emulator[self.key])
-        emulator_field[use_fortran_state] = state[self.key][use_fortran_state]
+        if self.fill_value is None:
+            emulator_field[use_fortran_state] = state[self.key][use_fortran_state]
+        elif isinstance(self.fill_value, str):
```

Comment:
What is the `str` case for?

Reply:
It's how I implemented the "no change" option. `LevelSlice(start=74, end=None, fill_value='air_temperature_input)` will replace the ml output with `air_temperature_input` at the specified levels.